RecoMuon/TrackingTools/interface/MuonTrackLoader.h

```diff
@@ -55,18 +55,18 @@ class MuonTrackLoader {
 
   /// Convert the trajectories into tracks and load the tracks in the event
   edm::OrphanHandle<reco::TrackCollection> loadTracks(
-      const TrajectoryContainer&, edm::Event&, const TrackerTopology& ttopo, const std::string& = "", bool = true);
```

Comment:
The containers elements are deleted by the function so strictly speaking this is not const.